### DISCUSSION TITLE
Vulkan: ImGui::Image throwing VK_ERROR_OUT_OF_POOL_MEMORY

### DISCUSSION THREAD

**Discussion by edvn0** [ocornut/imgui#6014] [New Users Build/Link/Run issues ONLY!]
Hi! I'm using ImGui::Image to render my scene into an imgui window (viewport). I use a descriptor pool of a 1000 combined imagesamplers.
My system is running at 144 fps, and after about 7 seconds (1000 / 144), vulkan claims that the descriptor pool is full, which to me seems reasonable. 

I tried to reset the command pool after each loop through my ImGui::Render code, but then the descriptor sets are invalid in relation to the new descriptor pool.

Have you run into this problem, and if so, how did you solve it? Thank you, and happy holidays!  🎅🏻

**Comment by Mirkopoj** [comment]
I had a similar problem, I was loading fonts each frame because it uses a comandbuffer.
But loading fonts once per buffer is enough, so I moved the code to the init stage and solved it.

Hope it helps.
